exercises/java/src/main/java/cards/Snap.java

```diff
@@ -13,10 +13,23 @@
     }
 
     public static void main(String[] args) {
+        printUsageInformation();
         Snap snap = new Snap(new AnimalDeck());
         snap.play();
     }
 
+    private static void printUsageInformation() {
```

Comment:
Apologies this snuck into this branch - the lack of instructions even confused us at first so thought it was worth adding.